blog-investigation/src/shared/components/buttons/PrimaryBtn.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Button } from "antd";
+import { Link } from "react-router-dom";
+
+import Translator from "../Translator";
+
+function PrimaryBtn({ linkTo, btnText }) {
+  return (
+    <Button type="primary">
+      <Link to={linkTo}>
```

Comment:
wrap Button with Link to improve UX - user will be able to click in any place of the button, otherwise only on text